src/utils/types.js

```diff
@@ -19,7 +19,7 @@ export const renderType = (type, index) => {
 export const jsonataTypeParser = `
 	$IsUnion := type = "UnionType";
 	$quote := function($val) { "'" & $val & "'" };
-	$GetNameExp := function($type) { $append($append($append($append($type[type="NameExpression"].name, $type[type="NullLiteral"] ? ['null'] : []), $type[type="AllLiteral"] ? ['Any'] : []), $map($type[type="StringLiteralType"].value, $quote)), $type[type="RecordType"] ? ['Object'] : []) };
+	$GetNameExp := function($type) { $append($append($append($append($append($type[type="NameExpression"].name, $type[type="NullLiteral"] ? ['null'] : []), $type[type="AllLiteral"] ? ['Any'] : []), $map($type[type="StringLiteralType"].value, $quote)), $type[type="RecordType"] ? ['Object'] : []), $type[type="ArrayType"] ? ['Array'] : []) };
```

Comment:
There are tough to read!

Reply:
they were tough to make, they should be tough to read!

Reply:
Seriously, though, it has gotten ridiculous.  I'll see if there's a way to refactor this.